SEED-platform/seed
# Update list of BETTER's measures returned

#### Any background context you want to provide?
Somewhere in the recent past, the list of measures from BETTER was updated. This includes adding a few new ones and renaming/removing others. Existing databases will still have the old fields but new runs in the old database will add in the new ones.

List of measures were taken from looking at this dict: https://github.com/LBNL-ETA/better_lbnl/blob/756f101292083bb053af06fd152ea7e1135878df/better_lbnl/building_analytics/models.py#L549

#### What's this PR do?
* Update the list of measures from BETTER

#### How should this be manually tested?
* Create a new property with meter data (import the better test data file)
* Select a property and run better
* Make sure the new measures are in the columns
* 

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate)

<img width="502" alt="image" src="https://github.com/SEED-platform/seed/assets/1907354/d3da844c-8d79-48f3-a92c-7865244f19a5">

<img width="468" alt="image" src="https://github.com/SEED-platform/seed/assets/1907354/f2cfb37f-0d2e-4824-9fac-0cc149c5f401">

Note that the previous results were also 0 and 1.
<img width="586" alt="image" src="https://github.com/SEED-platform/seed/assets/1907354/e83ae07c-5b90-4ad8-94fb-5f337abbe603">
